install/kubernetes/helm/istio/charts/sidecar-injector/templates/deployment.yaml

```diff
@@ -1,3 +1,4 @@
+{{- if .Values.global.sidecarInjectEnabled }}
```

Comment:
Is this a new parameter that we are introducing? How do I trigger this?

Reply:
https://github.com/istio/istio/pull/3010 - pls comment in there (sorry I had 2 change sets together), this is a helm param in values.yaml file.